frontend/device/kobo/device.lua

```diff
@@ -722,6 +725,45 @@ end
 
 function Kobo:getUnexpectedWakeup() return unexpected_wakeup_count end
 
+local function writeToSys(val, file)
+    local f = io.open(file, "we")
+    if not f then
+        logger.info("Cannot open:", file)
+        return
+    end
+    local re, err_msg, err_code = f:write(val, "\n")
+    if not re then
+        logger.err("Error writing to value to file:", val, file, err_msg, err_code)
+    end
+    f:close()
+    return re
+end
+
+--- put device into standby, touchscreen is enabled
+-- deadline ... deadline for standby
+function Kobo:standby(deadline)
+    -- just for wakup, dummy function
+    local function dummy() end
+
+    if deadline then
+        self.wakeup_mgr:addTask(deadline, dummy)
+    end
+
+    local TimeVal = require("ui/timeval")
+    local suspend_time_btv = TimeVal:boottime()
+
+    local re = writeToSys("standby", "/sys/power/state")
```

Comment:
Here there's no context. It's illegible and hard to follow, imho.
```suggestion
    local return_value = writeToSys("standby", "/sys/power/state")
```